packages/hds/src/components/Gauge/Gauge.tsx

```diff
@@ -0,0 +1,34 @@
+import { type HTMLAttributes, forwardRef } from 'react';
+import * as styles from './Gauge.css';
+
+export interface GaugeProps extends HTMLAttributes<HTMLDivElement> {
+  /** 게이지 바 연한 컬러 */
+  bgColor?: string;
+  /** 게이지 컬러 */
+  gauageColor?: string;
+  /** 게이지에 표시할 퍼센테지 @default 0 */
+  progress?: number;
+}
+
+const Gauge = forwardRef<HTMLDivElement, GaugeProps>(
+  ({ bgColor, gauageColor, progress = 0, ...rest }, ref) => {
+    return (
+      <div
+        className={styles.guageBackground}
+        style={{ backgroundColor: bgColor, width: '100%' }}
```

Comment:
```suggestion
        style={{ backgroundColor: bgColor, ..style }}
```



![image](https://github.com/HeeHeePresso/heehee-fe/assets/23569208/df649daa-0751-4a2e-8c36-a4da7ac4b4d5)

1. 중복입니다. styles.guageBackground 에 width 를 변경하셔야겠네요.
2. style 은 rest 에서 따로 빼서 전달해주세요(예시참고)
지금처럼 해두시면 style 객체를 prop 으로 넘겼을 때 정의해둔 backgroundColor 가 사라질 수 있습니다.

**설명**
```
< Gauage  ... style={{marginTop: 0}} /> // 이런식으로 사용할 때
< div  ... style={{backgroundColor: bgColor, width: '100%', marginTop: 0}} /> // 이렇게 합쳐져야하는데
< div  ... style={{marginTop: 0}} /> // 현재 코드는 이런식으로 합쳐짐
```